docs/references/javascript/organization/organization.mdx

```diff
@@ -16,8 +16,10 @@ This is a class that represents an organization. It's returned from the [`create
 | `slug` | `string \| null` | The organization slug. If supplied, it must be unique for the instance. |
 | `logoUrl` | `string \| null` | The full URL for the organization logo. (deprecated in favor of `imageUrl`) |
 | `imageUrl` | `string` | Holds the organization logo or default logo. |
+| `hasImage` | `boolean` | A getter boolean to check if the organization has a uploaded image. `false` if Clerk is displaying an avatar for the organization. |
```

Comment:
I think this prop comes up other times in the docs, we should update those instances to use this description as well.

```suggestion
| `hasImage` | `boolean` | A getter boolean to check if the organization has a uploaded image. Returns `false` if Clerk is displaying an avatar for the organization. |
```